resources/views/404.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts.app')
```

Comment:
Move this file into a new folder called `errors` so the file path will be: `resources/views/errors/404.blade.php`